plugins/woocommerce-blocks/tests/e2e/utils/editor/editor-utils.page.ts

```diff
@@ -417,9 +406,6 @@ export class EditorUtils {
 			await this.admin.visitSiteEditor( {
 				path: `/${ templateType }/all`,
 			} );
-			await this.page.goto(
-				`/wp-admin/site-editor.php?path=/${ templateType }/all`
-			);
```

Comment:
It's not directly related to this PR, but since I've just noticed it, I'm going to take the opportunity to remove it as it does exactly the same thing as the `visitSiteEditor` call above.